abnormal_security/icon_abnormal_security/util/api.py

```diff
@@ -64,6 +71,16 @@ def send_request(self, method: str, path: str, params: dict = None, payload: dic
         except requests.exceptions.HTTPError as e:
             raise PluginException(preset=PluginException.Preset.UNKNOWN, data=e)
 
+    def generate_filter_params(self, from_date: str = None, to_date: str = None) -> dict:
+        params = {}
+        if from_date or to_date:
```

Comment:
Check this out: 

```parsed_time = dateparser.parse(time_string)```

https://dateparser.readthedocs.io/en/latest/

Use that to parse dates, then convert them over to the date you need. It's much easier on the user.

Reply:
Thanks @jmcadams-r7, I've updated the code to use the dateparser. As this code was used by another action I've tested that too. See tests in the comment below :)